tox.ini

```diff
@@ -8,6 +8,16 @@ envlist =
     readme
 skip_missing_interpreters = True
 
+[python-cli-options]
+byte-warnings = -b
+byte-errors = -bb
+max-isolation = -E -s -I
+# some-isolation = -I
+# FIXME: Python 2 shim. Is this equivalent to the above?
+some-isolation = -E -s
+warnings-to-errors = -Werror
```

Comment:
I have somewhat mixed feelings about this block of config. On the one hand, I can see how it documents the short flags we use. And I didn't know you could have a custom section in `tox.ini` loaded like this, which is neat!

But I also find that it makes the usage sites _harder_ to read in some sense, since you have to lookup this table to figure out what the command is. (adding a bit of indirection)
The comparison is between
```
{envpython} \
    -I -bb -Werror
```
and
```
{envpython} \
    {[python-cli-options]byte-errors} \
    {[python-cli-options]max-isolation} \
    {[python-cli-options]warnings-to-errors}
```

I'm okay with it, at least to try out, but I'm just noting that I'm not fully convinced as of yet.